build/dependencies.props

```diff
@@ -40,7 +40,8 @@
 
     <!-- The Npqsql driver for EF Core 3.0 doesn't exist yet -->
     <MicrosoftEntityFrameworkCoreSqlServerVersion30>2.2.0</MicrosoftEntityFrameworkCoreSqlServerVersion30>
-    <MicrosoftAspNetCoreLibuvPackageVersion30>3.0.0-preview-18577-0035</MicrosoftAspNetCoreLibuvPackageVersion30>
+    <MicrosoftAspNetCoreLibuvPackageVersion30>3.0.0-preview-19059-0308</MicrosoftAspNetCoreLibuvPackageVersion30>
+    <MicrosoftAspNetCoreMvcNewtonsoftJson30> 3.0.0-preview-19059-0308</MicrosoftAspNetCoreMvcNewtonsoftJson30>
```

Comment:
We could use `$BenchmarksAspNetCoreVersion` as packages are released together. If fallback to a static version if it's not defined. I can do the change on the PR if you are ok with it.

Same for Libuv.